travis_specific/install-bin.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
Do we really want to have this file here instead of downloading it every time? If so, why is it necessary? I'd rather fetch it so we remain compatible with future Fabric releases.

Reply:
This file has been deleted since then. The latest version doesn't have it.